modules/auxiliary/dos/http/ibm_lotus_notes2.rb

```diff
@@ -0,0 +1,70 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name'           => "IBM Notes Denial Of Service",
+        'Description'    => %q(
+          This module exploits a vulnerability in the native browser that comes with IBM Lotus Notes.
+          If successful, the browser will crash after viewing the webpage.
+        ),
+        'License'        => MSF_LICENSE,
+        'Author'         => [
+          'Dhiraj Mishra',
+        ],
+        'References'     => [
+          [ 'EXPLOIT-DB', '42604'],
```

Comment:
```
['EDB', '42604'],
```

Reply:
Cool, @RootUp.  I restarted the Travis CI job (I think the previous failure was unrelated to your code, rather something with the Docker job).  If you want to update per @nixawk's suggestion (:+1:; looks like this slipped through on the previous PR --#8989-- as well) and cherry-pick or merge your documentation commit (#9125) to this branch, I think we'll be clear to land.

Reply:
Hi @pbarry-r7 i have made the necessary changes, I try merging #9125 but failed, could you please help me in cherry-pick  for the same.

Thank you !